### PR TITLE
ci: cleanup script cb

### PR DESCRIPTION
**Pull request by awsluja**
#### Description of changes
Cleanup script for Codebuild which focuses on stale resources, by @goldbez .
In short, this just runs the cleanup script without any parameters, so that it scans through each e2e test account and deletes old resources.
Successful run here https://tiny.amazon.com/kyjz1b29/IsenLink

#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies
- [ ] [Pull request labels](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#labels) are added

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by sobolk — APPROVED** [review]

File: codebuild_specs/aggregate_e2e_reports.yml [hunk 1]
```diff
@@ -10,9 +10,13 @@ phases:
       - ts-node ./wait-for-all-codebuild.ts $CODEBUILD_RESOLVED_SOURCE_VERSION ../$WAIT_FOR_IDS_FILE_PATH $PROJECT_NAME
       - cd ..
       - source ./shared-scripts.sh && _downloadReportsFromS3 $CODEBUILD_SOURCE_VERSION
+      - mkdir artifacts && echo "empty artifact" > artifacts/empty-artifact.txt
```
Comment (Amplifiyer) [inline codebuild_specs/aggregate_e2e_reports.yml:13]: Is this needed for cleanup script or just an unrelated change?

**Review by Amplifiyer — APPROVED** [review]
Looks good, one small question

File: codebuild_specs/aggregate_e2e_reports.yml (hunk 1 above)
Comment (awsluja) [reply codebuild_specs/aggregate_e2e_reports.yml:13]: just needed any artifact, as the cleanup job depends on the aggregate job to finish; Codebuild requires 'some' file to be produced in order to depend on a job
